src/Discord.Net.Core/Discord.Net.Core.csproj

```diff
@@ -1,14 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../Discord.Net.targets" />
-  <PropertyGroup>
-    <AssemblyName>Discord.Net.Core</AssemblyName>
-    <RootNamespace>Discord</RootNamespace>
-    <Description>The core components for the Discord.Net library.</Description>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
-  </PropertyGroup>
+  <Choose>
+    <When Condition=" '$(OS)' == 'Windows_NT' ">
```

Comment:
Why even do this in the first place? #RemoveNet45Support

Reply:
One can dream.